Dockerfile

```diff
@@ -1,117 +1,31 @@
-FROM ubuntu:16.04
+FROM girder/slicer_cli_web
 MAINTAINER Deepak Roy Chittajallu <deepk.chittajallu@kitware.com>
 
-# Install system pre-requisites
-RUN apt-get update && \
-    apt-get install -y \
-    build-essential \
-    wget \
-    git \
-    emacs vim \
-    make cmake cmake-curses-gui \
-    ninja-build && \
-    apt-get autoremove && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+# Insert commands to install any system pre-requisites and libraries here
 
-# Working directory
-ENV build_path=$PWD/build
+# Copy files of the plugin into the docker container
+ENV slicer_cli_web_plugin_path $build_path/slicer_cli_web_plugin
```

Comment:
Since `build_path` is set in the parent docker image, would it make sense to have it upper case + have a small comments at the top indicating which env variable are inherited and used ?